src/command_modules/azure-cli-eventgrid/setup.py

```diff
@@ -54,7 +54,10 @@
         'azure',
         'azure.cli',
         'azure.cli.command_modules',
-        'azure.cli.command_modules.eventgrid'
+        'azure.cli.command_modules.eventgrid',
+        'azure.cli.command_modules.eventgrid.sdk',
+        'azure.cli.command_modules.eventgrid.sdk.models',
+        'azure.cli.command_modules.eventgrid.sdk.operations'
```

Comment:
Yes so it was installed successfully.
Since the imports don't happen until runtime, it didn't catch it.